src/js/user-profile/helpers.jsx

```diff
@@ -76,3 +77,8 @@ export function isSIPEnabledForm(savedForm) {
   }
   return true;
 }
+
+export const disabledForms = {
+  '21P-527EZ': <div><span className="error">We’re sorry. We can’t give you access to your saved form right now. We’re working on ways to make it easier for you to apply for benefits online. Please check back later or apply by mail.</span><a href="/pension/pension-management-center/">Find out how to apply by mail</a>.</div>,
```

Comment:
The periods at the ends of the "Find out how to apply by mail" sentences are inconsistently placed, it might make sense to place them both after the link as that is how this is handled elsewhere.